### ISSUE TITLE
Copilot Status information for Usage not readable with JAWS for Windows 2025 virtual cursor

### ISSUE THREAD

**Issue by accesswatch** [microsoft/vscode#251836]
Type: <b>Bug</b>

I believe this may be a JAWS issue but hoping there might be another way of getting at this information to track usage data in real time and not through a report that takes 30 minutes to obtain. For example, is there a way to hook into an extension and get this data so that we can build something to show the data ourselves? The controls when using the tab key work just fine for jumping to the budget site or to manage Copilot features but I truly would like to get more real time data on the usage data that can be reviewed with a screen reader and not just spoken. I get the text but no way to review it by character. The virtual buffer is blank so hoping to write something myself to scrape this data.

VS Code version: Code 1.101.0 (dfaf44141ea9deb3b4096f7cd6d24e00c147a4b1, 2025-06-11T15:00:50.123Z)
OS version: Windows_NT x64 10.0.26100
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|13th Gen Intel(R) Core(TM) i5-1340P (16 x 2189)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.69GB (2.38GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (115)</summary>

Extension|Author (truncated)|Version
---|---|---
jelly-language-support|Act|1.2.2
Bookmarks|ale|13.5.0
mcp-store|aut|0.0.19
azurite|Azu|3.34.0
vscode-intelephense-client|bme|1.14.4
vscode-innosetup|cho|1.9.3
vscode-pandoc|Chr|0.4.8
promptboost|chr|0.8.0
npm-intellisense|chr|1.4.5
vscode-markdownlint|Dav|0.60.0
vscode-eslint|dba|3.0.10
dbcode|dbc|1.14.10
vscode-faker|dee|2.0.0
docker|doc|0.10.0
xml|Dot|2.5.1
gitlens|eam|17.2.0
prettier-vscode|esb|11.0.0
vscode-firefox-debug|fir|2.15.0
macros|ged|1.2.1
genaiscript-vscode|gen|1.142.8
codespaces|Git|1.17.3
copilot|Git|1.335.0
copilot-chat|Git|0.28.0
remotehub|Git|0.64.0
vscode-github-actions|git|0.27.2
vscode-pull-request-github|Git|0.110.0
geminicodeassist|goo|2.36.0
historian|h4c|1.3.4
vscode-htmlhint|HTM|1.10.0
copy-markdown-as-html|jer|1.1.0
mark-sharp|jon|1.8.1
language-matlab|Mat|1.3.3
markdown-shortcuts|mdi|0.12.0
rainbow-csv|mec|3.19.0
microsoft-testing|ms-|0.1.16
azure-dev|ms-|0.9.0
vscode-azure-github-copilot|ms-|1.0.26
vscode-azureappservice|ms-|0.26.2
vscode-azurecontainerapps|ms-|0.8.3
vscode-azurefunctions|ms-|1.17.3
vscode-azurelogicapps|ms-|5.94.22
vscode-azureresourcegroups|ms-|0.10.13
vscode-azurestaticwebapps|ms-|0.13.1
vscode-azurestorage|ms-|0.16.5
vscode-azurevirtualmachines|ms-|0.6.9
vscode-containers|ms-|2.0.3
vscode-cosmosdb|ms-|0.26.0
vscode-docker|ms-|2.0.0
csdevkit|ms-|1.20.35
csharp|ms-|2.80.16
vscode-dotnet-runtime|ms-|2.3.5
vscode-edge-devtools|ms-|2.1.9
data-workspace-vscode|ms-|0.6.3
mssql|ms-|1.32.1
sql-bindings-vscode|ms-|0.4.1
sql-database-projects-vscode|ms-|1.5.3
vscode-pgsql|ms-|1.4.2
autopep8|ms-|2025.2.0
black-formatter|ms-|2025.2.0
debugpy|ms-|2025.11.2025061301
pylint|ms-|2025.2.0
python|ms-|2025.9.2025061801
vscode-pylance|ms-|2025.6.1
vscode-python-envs|ms-|0.3.11691011
datawrangler|ms-|1.22.0
jupyter|ms-|2025.5.0
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.1.0
vscode-jupyter-cell-tags|ms-|0.1.9
vscode-jupyter-slideshow|ms-|0.1.6
remote-containers|ms-|0.418.0
remote-ssh|ms-|0.121.2025053015
remote-ssh-edit|ms-|0.87.0
remote-wsl|ms-|0.99.0
vscode-remote-extensionpack|ms-|0.26.0
azure-repos|ms-|0.40.0
live-server|ms-|0.4.15
powershell|ms-|2025.0.0
remote-explorer|ms-|0.6.2025051209
remote-repositories|ms-|0.43.2024112101
remote-server|ms-|1.6.2025061709
vscode-copilot-vision|ms-|0.2.2025032409
vscode-node-azure-pack|ms-|1.6.0
vscode-speech|ms-|0.16.0
vscode-typescript-next|ms-|5.9.20250617
vsliveshare|ms-|1.0.5948
windows-ai-studio|ms-|0.14.4
pytest-runner|pam|0.0.11
sqlite-viewer|qwt|0.10.6
html-snippets-pro|rit|1.0.1
file-bunny|rob|2.4.0
markdown-shortcuts|rob|0.7.1
markdown-snippets|rob|0.9.1
marky-dynamic|rob|0.4.2
marky-edit|rob|0.7.0
marky-markdown|rob|2.10.0
marky-stats|rob|0.11.0
snippets-ranger|rob|1.8.0
roo-cline|Roo|3.21.0
joplin-vscode-plugin|rxl|1.7.3
partial-diff|ryu|1.4.3
claude-dev|sao|3.17.14
markdown-preview-enhanced|shd|0.8.18
code-spell-checker|str|4.0.47
vscode-stripe|str|3.8.6
wav-preview|suk|2.6.0
vscode-ai-foundry|Tea|0.6.5
markdown-live-preview|Tec|1.5.0
intellicode-api-usage-examples|Vis|0.2.9
vscodeintellicode|Vis|1.3.2
gitdoc|vsl|0.2.3
wordpress-toolbox|wor|1.3.16
fake-data|wyc|1.0.0
php-debug|xde|1.36.1
markdown-all-in-one|yzh|3.6.3

</details>

**Comment by meganrogge** [comment]
Hi @accesswatch can you pls try with NVDA?

**Comment by accesswatch** [comment]
Yes, it can be read with NVDA. I am hoping to work around this by providing the information in an alternative way to ensure that users have access to it easily here. Is there an API to gather that data at the moment? I am willing to write an extension and upload it as a work around until the bug is fixed, again, this may be a vendor issue but I will be asking about this as well.

**Comment by meganrogge** [comment]
Thanks for your reply. I would suggest creating an issues for JAWS - as we cannot do anything about screen reader differences in  VS Code.
